packages/client/src/components/SafeSettings.js

```diff
@@ -450,6 +450,7 @@ function SafeSettings({ address, web3, name, threshold, safeOwners }) {
     proposeRemoveSigner,
   } = web3;
   const verifiedSafeOwners = safeOwners.filter((so) => so.verified);
+  const onlyOneOwnerLeft = verifiedSafeOwners.length === 1;
```

Comment:
i think we should generalize this to hide the `remove signer` button if `safeOwners.length == threshold`.  Because the safe will never let you have less signers than the threshold, it would be most useful to disable this button when removing a signer is not an option.

Reply:
That's a very good point, thanks for the insight!